tests/integration/components/add-reminder-test.js

```diff
@@ -0,0 +1,36 @@
+import { moduleForComponent, test, skip } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('add-reminder', 'Integration | Component | add reminder', {
+  integration: true,
+});
+
+test('it renders the add reminder form correctly', function(assert) {
+
+  this.render(hbs`{{add-reminder}}`);
+
+  assert.equal(this.$('label').first().text().trim(), 'Title:');
+  assert.equal(this.$('label:eq(1)').text().trim(), 'Date');
+  assert.equal(this.$('label:eq(2)').text().trim(), 'Notes');
+  assert.equal(this.$('input').length, 2);
+  assert.equal(this.$('textarea').length, 1);
+  assert.equal(this.$('button').length, 1);
+
+});
+
+skip('it should trigger addReminder action on submit', function(assert) {
+
+  this.set('fakeAdd', (store) => assert.deepEqual(store, {reminder: {title: 'grocery shopping', date: '', notes: ''}}))
+
+  this.render(hbs`{{add-reminder addReminder=(action fakeAdd)}}`);
+
+  this.$('.input-title').val('grocery shopping');
```

Comment:
Maybe try changing these from updating the value to setting the property. So this line would change to `this.set('title', 'grocery shopping'), and then yu wouldnt need the `.change()` line.